### PR TITLE
[IMPROVEMENT] Apply clang-format to all remaining files

### PR DESCRIPTION
**Pull request by canihavesomecoffee**
**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.
- [ ] **I have mentioned this change in the [changelog](https://github.com/CCExtractor/ccextractor/blob/master/docs/CHANGES.TXT).**

**My familiarity with the project is as follows (check one):**

- [ ] I have never used CCExtractor.
- [ ] I have used CCExtractor just a couple of times.
- [ ] I absolutely love CCExtractor, but have not contributed previously.
- [x] I am an active contributor to CCExtractor.

---

Applies clang-format to all failing files:
- GUI files
- lib_ccx files
- wrappers files

Excludes running checks on:
- third party (src/thirdparty)
- icons file of GUI

### CODE REVIEW

File: .github/workflows/format.yml [hunk 1]
```diff
@@ -18,5 +18,5 @@ jobs:
       - uses: actions/checkout@v1
       - name: Format code
         run: |
-          find src/ -type f -name '*.c' -name '*.c' | xargs clang-format -i
+          find src/ -type f -name '*.c' -not -path "src/thirdparty/*" -not -path "src/GUI/icon_data.c" | xargs clang-format -i
```
Comment (NilsIrl) [inline .github/workflows/format.yml]: Well I guess that's a way to do it :joy:

Comment (canihavesomecoffee) [reply .github/workflows/format.yml]: Well, we shouldn't touch third party, and that one file is something I'd rather not touch either for now. All the rest is neatly formatted from now on :)

**Comment by NilsIrl** [comment]
Why is the branch name "part1"?

File: .github/workflows/format.yml (hunk 1 above)
Comment (NilsIrl) [reply .github/workflows/format.yml]: We already don't touch third party: https://github.com/CCExtractor/ccextractor/blob/master/src/thirdparty/.clang-format

**Comment by canihavesomecoffee** [comment]
> Why is the branch name "part1"?

Because I intended to split it up and then changed my mind :)

File: .github/workflows/format.yml (hunk 1 above)
Comment (canihavesomecoffee) [reply .github/workflows/format.yml]: Ah, good remark on the config file for third party. I'll remove that.

**Pull request merged**